README.md

```diff
@@ -119,6 +119,23 @@ pip install -e .
 
 ### Running dot
 
+#### Graphical interface
+
+To use the graphical interface of dot run
+
+```bash
+conda activate dot
+python ./src/dot/ui/ui.py

```

Comment:
I suggest you update the setup.cfg to turn this into a CLI command.

for example:

```
dot-ui -h
```